gym/core.py

```diff
@@ -65,7 +65,10 @@ def render(
 
 
 decorator = _EnvDecorator
-if sys.version_info[0:2] == (3, 6):  # needed for https://github.com/python/typing/issues/449
+if sys.version_info[0:2] == (
+    3,
+    6,
+):  # needed for https://github.com/python/typing/issues/449
```

Comment:
If you move the comment to a new line, the if statement can be on a single line

Reply:
yeah, better